packages/react-core/src/components/ExpandableSection/examples/ExpandableSectionCustom.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ExpandableSection, Badge } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+export const ExpandableSectionCustom: React.FunctionComponent = () => {
```

Comment:
Nitpick: the file name + exported name could instead be `ExpandableSectionCustomToggle` to provide more context of the example.